setup.py

```diff
@@ -24,6 +24,11 @@
     packages=[
         "multipoint",
     ],
-    install_requires=["numpy>=1.16", "mpi4py>=3.0", "mdolab-baseclasses>=1.2.4"],
+    install_requires=[
+        "numpy>=1.16",
+        "mpi4py>=3.0",
+        "mdolab-baseclasses>=1.2.4",
+        "pyoptsparse>2.5.0"
```

Comment:
Should be 2.5.1 right?

Reply:
Either `>=2.5.1` or `>2.5.0` should work I think.

Reply:
oh yeah my bad